src/ui/AdvancedSearch/AdvancedSearch.ts

```diff
@@ -34,29 +35,46 @@ export class AdvancedSearch extends Component {
    * @componentOptions
    */
   static options: IAdvancedSearchOptions = {
+
     /**
-     * Specifies whether or not to include the built-in keywords section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Keywords** section.
+     *
+     * Default value is `true`.
      */
     includeKeywords: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies whether or not to include the built-in date section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Date** section.
+     *
+     * Default value is `true`.
      */
     includeDate: ComponentOptions.buildBooleanOption({ defaultValue: true }),
 
     /**
-     * Specifies whether or not to include the built-in document section.
-     * Default is `true`
+     * Specifies whether to include the built-in **Document** section.
+     *
+     * Default value is `true`.
      */
     includeDocument: ComponentOptions.buildBooleanOption({ defaultValue: true })
   };
 
+  /**
+   * The list of all available inputs inside the AdvancedSearch component.
```

Comment:
Thanks, I'll be more careful!

I'm taking a second glance at this entire changeset to make sure I didn't commit other stuff like that.

In the future, I'll add comments in my PR whenever I'm wondering whether I should expose something in the public documentation.